core/textile/bucket_factory.go

```diff
@@ -231,10 +231,6 @@ func (tc *textileClient) createBucket(ctx context.Context, bucketSlug string) (B
 		return b, nil
 	}
 
-	if err != nil {
```

Comment:
@perfectmak 

Why?

Reply:
It was useless... `err` before this statements is nil, so I just deleted it as redundant code

Reply:
@perfectmak 

True that! Confused by GitHub UI. 🙇🏻